src/algorithms/calorimetry/CalorimeterHitReco.cc

```diff
@@ -215,9 +215,14 @@ void CalorimeterHitReco::AlgorithmProcess() {
             cdim[0] = cell_dim[0];
             cdim[1] = cell_dim[1];
             m_log->debug("Using segmentation for cell dimensions: {}", fmt::join(cdim, ", "));
+        } else if (segmentation_type == "PolarGridRPhi2") {
+            cdim.resize(3);
+            cdim[0] = cell_dim[0];
+            cdim[1] = cell_dim[1];
```

Comment:
Makes no sense to implement this one. Downstream code expects things to be comparable in XYZ with cm as units.

Reply:
So is bounding box what we want? We should at least be explicit about it.

Reply:
No, we don't want it either, and likely we simply don't use the `dimensions` field for EndcapNHcal, at least in clusterer. This design is broken, I will be pushing to drop this eventually.

Reply:
But we should do something that doesn't cause an error message for every event. It's affecting ops.

Reply:
Didn't we hide the message into debug?

Reply:
Not the message for when the segmentation isn't recognized. That's still a warning, as it probably should be.